geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/AttributeIdxStrategyV5.scala

```diff
@@ -208,18 +210,23 @@ class AttributeIdxStrategyV5(val filter: QueryFilter) extends Strategy with Lazy
 @deprecated
 object AttributeIdxStrategyV5 extends StrategyProvider {
 
-  override def getCost(filter: QueryFilter, sft: SimpleFeatureType, hints: StrategyHints) = {
-    val cost = filter.primary.flatMap(getAttributeProperty).map { p =>
-      val descriptor = sft.getDescriptor(p.name)
-      val multiplier = if (descriptor.getIndexCoverage() == IndexCoverage.JOIN) 2 else 1
-      hints.cardinality(descriptor) match {
-        case Cardinality.HIGH    => 1 * multiplier
-        case Cardinality.UNKNOWN => 999 * multiplier
-        case Cardinality.LOW     => Int.MaxValue
-      }
-    }.sum
-    if (cost == 0) Int.MaxValue else cost // cost == 0 if somehow the filters don't match anything
-  }
+  override def getCost(sft: SimpleFeatureType,
+                       filter: QueryFilter,
+                       transform: Option[SimpleFeatureType],
+                       stats: GeoMesaStats,
+                       evaluation: CostEvaluation) =
+    AttributeIdxStrategy.getCost(sft, filter, transform, stats, evaluation)
+
+  // note: won't get called as we override getCost method, above
```

Comment:
well it uses the same logic as the newer table for the cost, so it just delegates to that. b/c I made the delegate cost methods protected, this class can't access them on the new attribute strategy class, so I had to override the main method - but still had to implement the delegate methods to satisfy the trait. Kind of smelly, but as you said it's deprecated so I don't mind it so much.